canvas.js

```diff
@@ -0,0 +1,214 @@
+let ctx;
+import {eventhandler} from "./Classes/EventClasses.js";

```

Comment:
not present in the code.

Reply:
sir, i forget to comment that line in the code but right now I updated it